### ISSUE TITLE
'cinnamon-settings themes' vacuums all comments from ~/.config/gtk-3.0/gtk.css

### ISSUE THREAD

**Issue by bjd-pfq** [linuxmint/cinnamon#8946]
Cinnamon 4.2.4

No LSB modules are available.
Distributor ID:	LinuxMint
Description:	Linux Mint 19.2 Tina
Release:	19.2
Codename:	tina

**Issue**
When running 'cinnamon-settings themes' from the cli, and playing with the new feature to adjust scrollbar width, I found out that the program overwrites my ~/.config/gtk-3.0/gtk.css; it takes its bare css settings, adds to them those of the scrollbar, and saves it. Gone are my precious comments and commented out trials of foregone days.
In short, the adjustment of the scrollbar is a quick, stealthy and dirty hack.

**Comment by icarter09** [comment]
@bjd-pfq is this a bug ticket or a feature request? If it's a feature request, what is it that you would like to see implemented in future Cinnamon versions?

**Comment by leigh123linux** [comment]
The issue is reproducible here
